package.json

```diff
@@ -15,33 +15,33 @@
     "url": "https://github.com/taskcluster/taskcluster-auth.git"
   },
   "dependencies": {
-    "assume": "^1.3.1",
-    "aws-sdk": "^2.149.0",
```

Comment:
in the time between the last commit and this one, they ugpraded? :)

Reply:
Wow... looks like they did!